src/mavedb/models/controlled_keyword.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+from sqlalchemy import Boolean, Column, Date, Integer, String, UniqueConstraint
+
+from mavedb.db.base import Base
+
+
+class ControlledKeyword(Base):
+    __tablename__ = 'controlled_keywords'
+
+    id = Column(Integer, primary_key=True, index=True)
+    key = Column(String, nullable=False)
+    value = Column(String, nullable=False)
```

Comment:
controlled_keyword is used for Keywords. They're the options for users to choose, not the users' uploading keywords. key and value form a unique pair. Only users uploaded keywords are allowed None value now. If users don't choose anything from front end, the keywords dictionary will be like {key: "AAA", value: None}. View model's value is Optional. Otherwise, it'll get an error if users don't choose anything for that keyword and that's why I comment value's validator either. They're temporary. They'll be all compulsory when we get the final list.